helper-cli/src/main/kotlin/commands/ImportPathExcludesCommand.kt

```diff
@@ -0,0 +1,112 @@
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.CommandWithHelp
+import com.here.ort.helper.common.findFilesRecursive
+import com.here.ort.helper.common.findRepositoryPaths
+import com.here.ort.helper.common.replacePathExcludes
+import com.here.ort.helper.common.sortPathExcludes
+import com.here.ort.helper.common.writeAsYaml
+import com.here.ort.model.config.PathExclude
+import com.here.ort.model.config.RepositoryConfiguration
+import com.here.ort.model.readValue
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+
+import java.io.File
+import java.lang.Exception
+
+@Parameters(
+    commandNames = ["import-path-excludes"],
+    commandDescription = "Import path excludes by repository from a file into the given repository configuration."
+)
+internal class ImportPathExcludesCommand : CommandWithHelp() {
+    @Parameter(
+        names = ["--path-excludes-file"],
+        required = true,
+        order = PARAMETER_ORDER_MANDATORY
+    )
+    private lateinit var pathExcludesFile: File
+
+    @Parameter(
+        names = ["--source-code-dir"],
+        required = true,
+        order = PARAMETER_ORDER_MANDATORY
+    )
+    private lateinit var sourceCodeDir: File
+
+    @Parameter(
+        names = ["--repository-configuration-file"],
+        required = true,
+        order = PARAMETER_ORDER_MANDATORY
+    )
+    private lateinit var repositoryConfigurationFile: File
+
+    override fun runCommand(jc: JCommander): Int {
+        val allFiles = findFilesRecursive(sourceCodeDir)
+
+        @Suppress("TooGenericExceptionCaught")
+        val repositoryConfiguration: RepositoryConfiguration = try {
+            repositoryConfigurationFile.readValue()
+        } catch (e: Exception) { RepositoryConfiguration() }
+
+        val existingPathExcludes = repositoryConfiguration.excludes?.paths ?: emptyList()
+        val importedPathExcludes = importPathExcludes().filter { pathExclude ->
+            allFiles.any { pathExclude.matches(it) }
+        }
+
+        val pathExcludes = (importedPathExcludes + existingPathExcludes).distinctBy { it.pattern }
+
+        repositoryConfiguration
+            .replacePathExcludes(pathExcludes)
+            .sortPathExcludes()
+            .writeAsYaml(repositoryConfigurationFile)
+
+        return 0
+    }
+
+    private fun importPathExcludes(): List<PathExclude> {
+        println("Analyzing $sourceCodeDir...")
+        val repositoryPaths = findRepositoryPaths(sourceCodeDir)
+        println("Found ${repositoryPaths.size} repositories in ${repositoryPaths.values.sumBy { it.size }} locations.")
+
+        println("Loading $pathExcludesFile...")
+        val pathExcludes: Map<String, List<PathExclude>> = pathExcludesFile.readValue()
```

Comment:
The type declaration could be omitted.

Reply:
It can only be moved, otherwise type inference fails. Is this what you asked for? 
...now using the better type.

Reply:
For consistency, better use:

```kotlin
val globalPathExcludes = pathExcludesFile.readValue<RepositoryPathExcludes>()
```